avg_age.js

```diff
@@ -0,0 +1,81 @@
+var mongoose = require("mongoose");
+var passwordHash = require("password-hash");
+
+var conn = mongoose.createConnection(
+  "mongodb://localhost:27017/userdata",
+  {
+    useNewUrlParser: true,
+    useFindAndModify: true,
+    useCreateIndex: true,
+    useUnifiedTopology: true,
+  },
+  function (err, db) {
+    if (err) {
+      console.log("no");
+    }
+  }
+);
+
+var usersprofile_schema = mongoose.Schema(
+  {
+    dob: {
+      type: String,
+      required: true,
+    },
+    mobile: {
+      type: String,
+      required: true,
+    },
+    address: {
+      type: String,
+      required: true,
+    },
+  },
+  {
+    strict: false,
+    collection: "userdatanew",
+  }
+);
+var user = conn.model("userdatanew", usersprofile_schema);
```

Comment:
same code in many file so create a specific file or use one time

Reply:
import variable userprofile_schema from a file.